src/FormattingToolbar/index.js

```diff
@@ -274,15 +339,16 @@ export default class FormatToolbar extends React.Component {
         position='bottom center'
         trigger={
           <ToolbarIcon
+            ref={this.linkButtonRef}
             aria-label={type}
             background={bgActivity}
             width={wi}
             height={hi}
             padding={pa}
             viewBox={vBox}
             className={classInput}
-            onClick={event => action.onClickLink(event, this.props.editor)}>
-              {icon(fillActivity)}
+            onClick={() => this.onClickLinkButton()}>
```

Comment:
this can simply be `onClick={this.onClickLinkButton}`

Reply:
Can't. Because it not arrow function. There is `this` usage in that method.
Currently this project not support arrow function in class.